tasks/dials.ts

```diff
@@ -221,10 +221,11 @@ task("dials-snap", "Snaps Emissions Controller's dials")
 
```

Comment:
it is just styling but i would rather see a filter latestDialVotes.filter().forEach
than a return inside the foreach

Reply:
Not really possible on that array without creating mismatched indexes, but reworked to add a filter where the string is built.